hazelcast/src/test/java/com/hazelcast/jet/SimpleTestInClusterSupport.java

```diff
@@ -119,10 +118,13 @@ public static void supportAfterClass() throws Exception {
             if (factory != null) {
                 SUPPORT_LOGGER.info("Terminating instance factory in SimpleTestInClusterSupport.@AfterClass");
                 spawn(() -> factory.terminateAll())
-                        .get(5, TimeUnit.SECONDS);
+                        .get(1, TimeUnit.MINUTES);
             }
-        } catch (TimeoutException e) {
-            SUPPORT_LOGGER.warning("Terminating instance factory timed out", e);
+        } catch (Exception e) {
+            // Log the exception, so it is visible in log file for the test class,
+            // otherwise it is only visible in surefire test report
+            SUPPORT_LOGGER.warning("Terminating instance factory failed", e);
```

Comment:
We don't need to log this error, it will be logged with the test failure.

Reply:
The stacktrace is be part of the in the xml report file, but not in the log file, Logging it directly provides the exact point when the exception occurs, which is useful for investigation of any test failures.